common/static/common/templates/components/paging-header.underscore

```diff
@@ -2,10 +2,10 @@
     <div class="meta">
         <%= messageHtml %>
     </div>
-    <nav class="pagination pagination-compact top">
+    <nav class="pagination pagination-compact top" aria-label="Compact Pagination">
         <ol>
-            <li class="nav-item previous"><a class="nav-link previous-page-link" href="#"><i class="icon fa fa-angle-left"></i> <span class="nav-label"><%= gettext("Previous") %></span></a></li>
-            <li class="nav-item next"><a class="nav-link next-page-link" href="#"><span class="nav-label"><%= gettext("Next") %></span> <i class="icon fa fa-angle-right"></i></a></li>
+            <li class="nav-item previous"><button class="nav-link previous-page-link"><i class="icon fa fa-angle-left"></i> <span class="nav-label"><%- gettext("Previous") %></span></button></li>
+            <li class="nav-item next"><button class="nav-link next-page-link"><span class="nav-label"><%- gettext("Next") %></span> <i class="icon fa fa-angle-right"></i></button></li>
```

Comment:
Scratch that - reverting this change, as it does not have anything to do with teams topics.